website_sale_infinite_scroll/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,3 @@
+To select different preload go to * General Settings > Website:
+ - Enabled "Website sale infinite scroll preloader"
+ - Select a new gif file
```

Comment:
Is this still valid?

Reply:
yes)

Reply:
![image](https://github.com/OCA/e-commerce/assets/43703214/5df2f3a7-3345-4d9e-9f30-5ec1fd0d1155)
Okay, I fixed it